scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -35,16 +34,23 @@ class ScoveragePlugin(val global: Global) extends Plugin {
         options.dataDir = opt.substring("dataDir:".length)
       } else if (opt.startsWith("extraAfterPhase:") || opt.startsWith("extraBeforePhase:")) {
         // skip here, these flags are processed elsewhere
-      } else {
+      } else if (opt.startsWith("useEnvironment:")) {
+        options.useEnvironment = opt.substring("useEnvironment:".length).toBoolean
+      } else{
         error("Unknown option: " + opt)
       }
     }
-    if (!opts.exists(_.startsWith("dataDir:")))
-      throw new RuntimeException("Cannot invoke plugin without specifying <dataDir>")
+
+    if (opts.exists{p: String => p.contains("useEnvironment") && p.contains("false")} || (!opts.exists(_.startsWith("useEnvironment")))) {
+      if (!opts.exists(_.startsWith("dataDir:")))
+        throw new RuntimeException("Cannot invoke plugin without specifying <dataDir>")
```

Comment:
Or without specifying new option?

Reply:
that is being checked by `!opts.exists(_.startsWith("useEnvironment"))`